build/validation/Containerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.11-9 AS forklift-opa
```

Comment:
Ahh validations, we used to download the whole project and build it manually, no longer needed as we have accessible the internet and can just download the ` opa` binary.
This could be enough:

```
FROM registry.access.redhat.com/ubi9-minimal:9.4-1227.1725849298
RUN microdnf -y update && microdnf -y clean all
RUN curl https://github.com/open-policy-agent/opa/releases/download/v0.62.1/opa_linux_amd64 -o /usr/bin/opa
COPY validation/policies /usr/share/opa/policies/
COPY validation/entrypoint.sh /usr/bin/

ENTRYPOINT ["/usr/bin/entrypoint.sh"]

LABEL \
        com.redhat.component="forklift-validation-container" \
        version="$CI_VERSION" \
        name="forklift/forklift-validation-rhel9" \
        license="Apache License 2.0" \
        io.k8s.display-name="Forklift" \
        io.k8s.description="Forklift - Validation Service" \
        io.openshift.tags="migration,mtv,forklift" \
        summary="Forklift - Validation Service" \
        maintainer="Forklift by Konveyor Community <forklift-dev@googlegroups.com>"
```